### DISCUSSION TITLE
suspend_mode storage location of .save files

### DISCUSSION THREAD

**Discussion by Jeansen** [vagrant-libvirt/vagrant-libvirt#1705] [Q&A]
I wonder how I can set the storage location for `suspend_mode=mangagedsaved` .save files. Currently, they are stored in `/var/lib/libvirt/qemu/save/`. How can I change that?

**Comment by electrofelix** [comment]
I can't find anything, so I suspect that this is the difference between `save` and `managedsave`, in the former you can control where the state is saved to, while in the latter it's managed by libvirt. I can't even see that there is an option to control it.

Looking at the upstream test code, I can see that they have tests around the path for the image when testing https://github.com/libvirt/libvirt/blob/608c4b249e8b683b311d8b0205f23fdd6082d06a/src/test/test_driver.c#L2267, but I couldn't find a similar test for managedsave behaviour, so I suspect it's not possible to control the output path.

**Reply by Jeansen** [reply]
Yes, that's right. Using save you can give it a destination. But the vagrant-libvirt plugin only offers `mangagedsave` and `pause`. There is no `save` as a third option for `suspend_mode`.
